docs/tutorials/getting-started.md

```diff
@@ -81,4 +81,4 @@ Discourse will be deployed with `discourse-k8s` as default hostname. In order to
 
 After that, visit `http://discourse-k8s` to reach Discourse.
 
-This charm allows you to add a admin user by executing the corresponding action. Instead of interacting with the Discourse UI just run `juju run-action discourse-k8s/0 add-admin-user email=email@example.com password=somepwd --wait` and it will be registered and validated.
+This charm allows you to add a admin user by executing the corresponding action. Instead of interacting with the Discourse UI just run `juju run-action discourse-k8s/0 add-admin-user email=email@example.com password=somepassword --wait` and it will be registered and validated.
```

Comment:
should this be `<secure password>` to indicate that it should be replaced by a secure password?

Reply:
Good point, could be - I was replacing `somepwd` with `somepassword` as following the guide produces an error (somepasswd < 10 characters and the action fails). So `somepassword` works as an example. I could place `<secure password>`, though I need to also point out that it has to also be more than 10 characters. What do you think?

Reply:
Yes that is a good approach. It is important not to encourage setting insecure passwords in our tutorials